node/cn/backend.go

```diff
@@ -99,7 +104,8 @@ type CN struct {
 
 	components []interface{}
 
-	governance *governance.Governance
+	governance     *governance.Governance
+	stakingManager *reward.StakingManager
 }
 
 func (s *CN) AddLesServer(ls LesServer) {
```

Comment:
Method description comment is missing.

Reply:
I'll make a new pr for method description comments